tests/text/pipelines/test_multifield_bimpm.py

```diff
@@ -34,59 +33,109 @@ def training_data_yaml(tmpdir):
     return str(yaml_file)
 
 
-@pytest.fixture
-def pipeline_yaml(tmpdir):
-    yaml_dict = {
-        "type": "multifield_bimpm",
-        "pipeline": {
-            "token_indexers": {"tokens": {"type": "single_id"}},
-            "as_text_field": False,
-        },
-        "architecture": {
-            "text_field_embedder": {
-                "tokens": {
-                    "type": "embedding",
-                    "padding_index": 0,
-                    "embedding_dim": 10,
-                }
+@pytest.fixture(params=["singlefield", "multifield"])
+def pipeline_yaml(tmpdir, request):
+    pipeline_dict = {
+        "name": "biome-bimpm",
+        "tokenizer": {"text_cleaning": {"rules": ["strip_spaces"]}},
+        "features": {
+            # "tokens": {
+            #     "indexer": {"type": "single_id", "lowercase_tokens": True},
+            #     "embedder": {
+            #         "type": "embedding",
+            #         "embedding_dim": 32,
+            #         "padding_index": 0,
+            #     },
+            # },
+            # Missing: padding_index
+            "words": {
+                "embedding_dim": 32,
+                "lowercase_tokens": True,
+                "trainable": True,
+            },
+            # "token_characters": {
+            #     "indexer": {
+            #         "type": "characters",
+            #         "character_tokenizer": {"lowercase_characters": True},
+            #     },
+            #     "embedder": {
+            #         "type": "character_encoding",
+            #         "embedding": {"embedding_dim": 32, "padding_index": 0},
+            #         "encoder": {
+            #             "type": "gru",
+            #             "input_size": 32,
+            #             "hidden_size": 112,
+            #             "num_layers": 1,
+            #             "bidirectional": True,
+            #         },
+            #         "dropout": 0.2,
+            #     },
+            # },
+            # Missing: lowercase_characters, padding_index
```

Comment:
`lowercase_characters` should be included as a top level config param in `CharFeaturesSpec` as it is sufficiently common, same with others you consider highly relevant. Less frequent configs can still be included in configuration time using `embedder` and `indexer` sections in this block (we will document this)

Reply:
done